multisig/proposals/arbitrumsepolia/vip-015/index.ts

```diff
@@ -0,0 +1,223 @@
+    },
+    {
+      target: arbitrumsepolia.RESILIENT_ORACLE,
+      signature: "setTokenConfig((address,address[3],bool[3]))",
+      params: [
+        [
+          Mock_wstETH,
+          [
+            arbitrumsepolia.CHAINLINK_ORACLE,
+            "0x0000000000000000000000000000000000000000",
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
+    {
+      target: arbitrumsepolia.CHAINLINK_ORACLE,
+      signature: "setDirectPrice(address,uint256)",
+      params: [Mock_wstETH, parseUnits("1.1", 18)],
+    },
+    { target: COMPTROLLER_LIQUID_STAKED_ETH, signature: "acceptOwnership()", params: [] },
+    {
+      target: COMPTROLLER_LIQUID_STAKED_ETH,
+      signature: "setPriceOracle(address)",
+      params: [arbitrumsepolia.RESILIENT_ORACLE],
+    },
+    {
+      target: arbitrumsepolia.POOL_REGISTRY,
+      signature: "addPool(string,address,uint256,uint256,uint256)",
+      params: [
+        "Liquid Staked ETH",
+        COMPTROLLER_LIQUID_STAKED_ETH,
+        parseUnits("0.5", 18),
+        parseUnits("1.02", 18),
+        parseUnits("100", 18),
+      ],
+    },
+    {
+      target: arbitrumsepolia.VTREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
+      params: [Mock_wstETH, parseUnits("2", 18), arbitrumsepolia.GUARDIAN],
+    },
+    {
+      target: Mock_wstETH,
+      signature: "approve(address,uint256)",
+      params: [arbitrumsepolia.POOL_REGISTRY, 0],
+    },
+    {
+      target: Mock_wstETH,
+      signature: "approve(address,uint256)",
+      params: [arbitrumsepolia.POOL_REGISTRY, parseUnits("2", 18)],
+    },
+    {
+      target: VwstETH,
+      signature: "setReduceReservesBlockDelta(uint256)",
+      params: ["86400"],
+    },
+    {
+      target: VwstETH,
+      signature: "setProtocolShareReserve(address)",
+      params: [PSR],
+    },
+    {
+      target: arbitrumsepolia.POOL_REGISTRY,
+      signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+      params: [
+        [
+          VwstETH,
+          parseUnits("0.93", 18),
+          parseUnits("0.95", 18),
+          parseUnits("2", 18),
+          arbitrumsepolia.VTREASURY,
+          parseUnits("8000", 18),
+          parseUnits("800", 18),
+        ],
+      ],
+    },
+    {
+      target: arbitrumsepolia.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[Mock_weETH, CHAINLINK_ETH_FEED, STALE_PERIOD_3M]],
+    },
+    {
+      target: arbitrumsepolia.RESILIENT_ORACLE,
+      signature: "setTokenConfig((address,address[3],bool[3]))",
+      params: [
+        [
+          Mock_weETH,
+          [
+            arbitrumsepolia.CHAINLINK_ORACLE,
+            "0x0000000000000000000000000000000000000000",
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
```

Comment:
The oracle for weETH should be a OneJumpOracle, with these properties:

Correlated token: weETH
Underlying token: ETH
Intermediate oracle: Chainlink. We will need to configure a fixed price for weETH on that oracle (i.e. 1.1)

Reply:
Configured oneJumpOracle for weETH, 9cd008c26bd5eba3b018ca161798f55d17d6dbdc
Deployment files for the oneJumpOracle: https://github.com/VenusProtocol/oracle/pull/214